packages/webapp/pages/notifications.tsx

```diff
@@ -91,6 +126,73 @@ const Notifications = (): ReactElement => {
 
   usePromotionModal();
 
+  const { show } = useContextMenu({ id: contextId });
```

Comment:
I personally would move this to a custom component, as we currently do for context menu's
Just in case it becomes general.

Reply:
Right. Let me move this and the `onOptionsClick` you mentioned.